requirements/specifications/metrics/media-metrics.md

```diff
@@ -0,0 +1,293 @@
+
+The remainder of this section outlines guidelines that apps should consider when calling `mediaSeeked`.
+
+The `mediaSeeked` API **SHOULD** be called when an "intent to seek" is successfully fulfilled and right before `mediaPlaying` is called due to the seek operation resulting in playback.
+
+The `mediaSeeked` API **SHOULD NOT** be called for any cancelled or ignored intents to seek, to ensure that every `mediaSeeked` correlates to the preceding `mediaSeeking` call.
+
+The `position` value **SHOULD** be a number greater than 0 and less than 1 denoting the percent of the content intended to seek to, if the duration of the content is static and known, e.g. VOD content.
+
+The `position` value **SHOULD** be an integer number of secionds from 1 to 86400 denoting the number of seconds from the "beginning" of the content intended to seek to, if the duration of the content is dynamic or not known, e.g. live content.
+
+The "beginning" of the content **SHOULD** be considered as the start of the broadcast/event/etc., if known, otherwise it **MAY** be the start of the available media, e.g available on a CDN.
+
+### 4.9. Rate Change
+The `Metrics` module **MUST** include a `mediaRateChanged` API for denoting a change to the playback rate, e.g. going from 1x to 2x playback speed.
+
+The `mediaRateChanged` API **MUST** have a required `entityId` parameter to specify what entity from the app catalog the seek is occuring in.
+
+The `mediaRateChanged` API **MUST** have a required `rate` parameter denoting the new playback rate.
+
+The remainder of this section outlines guidelines that apps should consider when calling `mediaRateChanged`.
+
+The `mediaRateChanged` API **SHOULD** be called when the playback rate of the content is explicitly changed for any reason, user-initiated or otherwise.
+
+The `mediaRateChanged` API **SHOULD** be called regardless of whether the rate change is due to the media pipeline rasterizing frames at a different rate, or due to a simulated rasterization, e.g. "trick play" with I-frames.
+
+### 4.10. Rendition Change
+The `Metrics` module **MUST** include a `mediaRenditionChanged` API for denoting a change to rendition, e.g. bitrate or dimension changes for quality.
+
+The `mediaRenditionChanged` API **MUST** have a required `entityId` parameter to specify what entity from the app catalog the seek is occuring in.
+
+The `mediaRenditionChanged` API **MUST** have a required `bitrate` number parameter denoting the new bitrate.
+
+The `mediaRenditionChanged` API **MUST** have a required `width` number parameter denoting the new width.
+
+The `mediaRenditionChanged` API **MUST** have a required `height` number parameter denoting the new height.
+
+The `mediaRenditionChanged` API **MUST** have an optional `profile` string parameter with a description of the new profile, e.g. 'HDR', etc.
```

Comment:
Should we also capture the reason of the change (auto vs user-selected resolution)?  That might be useful for identifying content where the user forces a better viewing experience than the auto algorithm chooses (I sometimes have to do this with YT).